### PR TITLE
add nft pallet

### PR DESCRIPTION
**Pull request by Satoshi-Kusumoto**
Currently we use not the latest version of NFT pallet (deposit upon class creation is missing) since it seems requires some refactor of the code to achieve this.
Part of #63

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -740,6 +740,25 @@ impl pallet_aura::Config for Runtime {
...
+}
+
+impl orml_nft::Config for Runtime {
+	type ClassId = u32;
+	type TokenId = u64;
+	type ClassData = pallet_nft::ClassData<BlockNumber, Self::ClassId>;
+	type TokenData = pallet_nft::TokenData;
+	type MaxClassMetadata = MaxClassMetadata;
+	type MaxTokenMetadata = MaxTokenMetadata;
+}
```
Comment (h4n0) [inline runtime/src/lib.rs:755]: What is the reason of adding orml nft into runtime?

Comment (suinuj) [reply runtime/src/lib.rs:755]: have the same question.

Comment (suinuj) [reply runtime/src/lib.rs:755]: need its configuration?

**Review by suinuj — APPROVED** [review]

File: runtime/src/lib.rs (hunk 1 above)
Comment (Satoshi-Kusumoto) [reply runtime/src/lib.rs:755]: It is a dependency of pallet NFT

**Pull request merged**